05/app.js

```diff
@@ -0,0 +1,41 @@
+function Student(name, lastname){
+    this.firstName = name;
+    this.lastName = lastname;
+    this.grades = {};
+}
+
+Student.prototype.addGrade = function(subject, grade){
+    if(typeof this.grades[subject] === 'undefined'){
+    this.grades[subject] = [];
+    };
+    this.grades[subject].push(grade);
+}
+
+Student.prototype.getAverageGrade = function(subject) {
+    // suma ocen
+    let result = 0;
+    if(subject) {
+        this.grades[subject].forEach(element => {
+            result+=element;
+        });
+        // średnia ocen z przedmiotu
+        result = result/this.grades[subject].length;
+        console.log(`Średnia ocen z ${subject} wynosi:`);
```

Comment:
Należało wykorzystać `return`, jeśli chcielibyśmy wyświetlić to pewnie funkcja nazywałaby się `show...`